modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -238,6 +243,9 @@ public static Variant.Type TypeToVariantType(Type type)
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal extern static double godot_icall_GD_randf_range(double from, double to);
 
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal extern static float godot_icall_GD_randfn();
```

Comment:
I know nothing about C# interop but aren't you forgetting the parameters here?

Reply:
Yeah, thanks